app/models/manageiq/providers/ovirt/inventory/parser/infra_manager.rb

```diff
@@ -113,6 +114,19 @@ def storagedomains
     end
   end
 
+  def advertised_images
+    collector.advertised_images.each do |image|
```

Comment:
Are the advertised images an ISO datastore?  I thought they would be IsoImages that belong to an IsoDatastore

Reply:
Yes you are correct they belong to an IsoDatastore. I had a brain fart moment here haha